src/components/Select.js

```diff
@@ -1,3 +1,4 @@
+const _ = require('lodash');
```

Comment:
@paniclater @cerinman I don't see either of those in the lodash node module folder - I actually looked for this yesterday because I noticed we did something similar in a different component. I just see the shallow comparison...

![screen shot 2016-03-15 at 9 50 09 am](https://cloud.githubusercontent.com/assets/13579578/13784061/6a8b4e10-ea93-11e5-840f-c6db23d771b4.png)